### ISSUE TITLE
Check any images have a description

### ISSUE THREAD

**Issue by Angelsaur** [GeorgiaHCA/AutSPACEs#47]
### Summary
You’ll need to check that any non-decorative images (including charts or diagrams) have an accompanying text description. That way, users can still access the relevant information even if they cannot see the image

### Deadline 
20/02/2021

### What needs to be done?

Take your sample pages and check that any images either:

have appropriate alternative (or ‘alt) text that explains the information the image conveys - you can usually check this in the content management system you use to update your website
are described in any surrounding body text
You do not need to check decorative images - these do not need an accompanying description.

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
